### PR TITLE
Improve the SDK and introduce a sample app with a web interface

### PR DESCRIPTION
**Pull request by thivi**
This PR improves the Node SDK and creates a sample app using Express.js and ejs template engine.

### CODE REVIEW

**Review by brionmario — APPROVED** [review]

File: samples/asgardeo-express-app/static/app.css [hunk 1]
```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (brionmario) [inline samples/asgardeo-express-app/static/app.css]:
```suggestion
 * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```

File: samples/asgardeo-express-app/static/images/node-js.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...7zm141.093 107.164c1.574-.914 2.543-2.602 2.543-4.422V91.21c0-1.824-.97-3.507-2.547-4.425l-42.1-24.44c-1.59-.92-3.54-.92-5.13 0l-42.36 24.45c-1.59.92-2.56 2.6-2.56 4.43v48.9c0 1.84.99 3.54 2.58 4.45l42.09 23.99c1.55.89 3.45.9 5.02.03l25.46-14.15c.8-.45 1.31-1.3 1.31-2.22 0-.92-.49-1.78-1.29-2.23l-42.62-24.46c-.8-.45-1.29-1.3-1.29-2.21v-15.34c0-.916.48-1.76 1.28-2.216l13.26-7.65c.79-.46 1.76-.46 2.55 0l13.27 7.65c.79.45 1.28 1.3 1.28 2.21v12.06c0 .91.49 1.76 1.28 2.22.79.45 1.77.45 2.56-.01zm0 0"/><path fill="#689f63" fill-rule="evenodd" d="M394.538 105.2c.3-.177.676-.177.98 0l8.13 4.69c.304.176.49.5.49.85v9.39c0 .35-.186.674-.49.85l-8.13 4.69c-.304.177-.68.177-.98 0l-8.125-4.69c-.31-.176-.5-.5-.5-.85v-9.39c0-.35.18-.674.49-.85zm0 0"/><g clip-path="url(#a)" transform="translate(-78.306 -164.016)"><path fill="url(#b)" d="M331.363 246.793l-118.715-58.19-60.87 124.174L270.49 370.97zm0 0"/></g><g clip-path="url(#c)" transform="translate(-78.306 -164.016)"><path fill="url(#d)" d="M144.07 264.004l83.825 113.453 110.86-81.906-83.83-113.45zm0 0"/></g><g clip-path="url(#e)" transform="translate(-78.306 -164.016)"><path fill="url(#f)" d="M197.02 225.934v107.43h91.683v-107.43zm0 0"/></g></svg>
```
Comment (brionmario) [inline samples/asgardeo-express-app/static/images/node-js.svg]: Missing EOF. Also better to format IMO

File: samples/asgardeo-express-app/views/index.ejs [hunk 3]
```diff
@@ -0,0 +1,276 @@
+<!--
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (brionmario) [inline samples/asgardeo-express-app/views/index.ejs]:
```suggestion
 * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```

File: samples/asgardeo-express-app/static/images/node-js.svg (hunk 2 above)
Comment (thivi) [reply samples/asgardeo-express-app/static/images/node-js.svg]: Resolved by [adc2221](https://github.com/asgardeo/asgardeo-auth-node-sdk/pull/7/commits/adc2221a814bf3ba2f0e927a4dfaaba5e42040cb)

**Pull request merged**